Resources/Textures/Objects/Devices/boombox.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by BeatusCrow P.S. The sprite is taken from the assembly of the Armored Warfare game KAG",
+  "size": {
+    "x": 16,
```

Comment:
Do you even have permission to use this or did you just take it from the game files? As that's not how copyright works.

Reply:
Good time of day. I seem to have forgotten to indicate in the description of the PR that this is not actually a finished project, but only a proposal that I am ready to finish if you like this idea. I don't have permission to use this sprite, but again, I added it so that at least some texture would be shown... I don't draw very well myself (you can see from the cassettes), if you don't set the status "DO NOT MERGE", then I'll ask my friend spriter to make textures. And about the songs... It would be nice if you could throw in a list of things worth adding. I would also like to hear any comment about the uplink. Is a new tab appropriate?

Reply:
Keep the PR small and make a basic boombox first before trying to add more as it's just going to take longer to get it merged.